src/main/java/spireMapOverhaul/BetterMapGenerator.java

```diff
@@ -99,6 +100,8 @@ public ArrayList<ArrayList<MapRoomNode>> generate(Random rng, int width, int hei
         } while (!finishMap(rng, planner, pathDensity));
         queueCommandZones.clear();
 
+        queueCommandZones.clear();
```

Comment:
This line duplicates the code immediately above it (which got added/changed after you started your zone). You can delete it.